tests/lib/rules/require-unicode-regexp.js

```diff
@@ -302,6 +321,144 @@ ruleTester.run("require-unicode-regexp", rule, {
                     }
                 ]
             }]
+        },
+        {
+            code: "/foo/",
+            options: [{ requireFlag: "v" }],
+            languageOptions: { ecmaVersion: 2024 },
+            errors: [{
+                messageId: "requireVFlag",
+                suggestions: [
+                    {
+                        messageId: "addVFlag",
+                        output: "/foo/v"
+                    }
+                ]
+            }]
+        },
+        {
+            code: "/foo/u",
+            options: [{ requireFlag: "v" }],
+            languageOptions: { ecmaVersion: 2024 },
+            errors: [{
+                messageId: "requireVFlag",
+                suggestions: [
+                    {
+                        messageId: "addVFlag",
+                        output: "/foo/v"
+                    }
+                ]
+            }]
+        },
+        {
+            code: "/foo/u",
+            options: [{ requireFlag: "v" }],
+            languageOptions: { ecmaVersion: 6 },
+            errors: [{
+                messageId: "requireVFlag",
+                suggestions: null
+            }]
+        },
+        {
+            code: "/[[a]/u",
+            options: [{ requireFlag: "v" }],
+            languageOptions: { ecmaVersion: 2024 },
+            errors: [{
+                messageId: "requireVFlag",
+                suggestions: null
+            }]
+        },
```

Comment:
right now these tests are failing because rule is providing suggestions for them.

Reply:
I'm not seeing any failing tests either locally or with the PR checks (?)

Reply:
@Tanujkanti4441  All tests seem to be passing in CI?

Reply:
Oh! i am really sorry i missed the utils file while testing the code.